packages/builders/src/components/ActionRow.ts

```diff
@@ -46,4 +47,11 @@ export class ActionRow<T extends ActionRowComponent = ActionRowComponent> extend
 			components: this.components.map((component) => component.toJSON()),
 		};
 	}
+
+	public equals(other: APIActionRowComponent<APIMessageComponent> | ActionRow) {
```

Comment:
does this really belong in builders? Thought it'd be something for discord.js so we can compare with Embeds from there 🤔

Reply:
I don't see why not? Could be useful outside of d.js

Reply:
Yeah but my question is if you will be able to throw the Embed class from discord.js into this and have it work.
I also just realized halfway through writing this that this method wasn't added to Embed so, please? 🥺

Reply:
> Yeah but my question is if you will be able to throw the Embed class from discord.js

Technically speaking yes, since d.js's extends builders class, it'll work just fine.
![image](https://user-images.githubusercontent.com/17960496/154118409-ac1284b7-fbe4-46db-928c-3b3c727a7a32.png)